client/src/pages/Overview.js

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import styled from "@emotion/styled";
+import { Link } from "react-router-dom";
+import { PagesContainer } from "../components/Container";
+import Cards from "../components/Cards";
+import { AddButton } from "../components/Button";
+
+const CustomLink = styled(Link)`
+  width: 100%;
+  height: 100%;
+  text-decoration: none;
+  color: ${({ theme }) => theme.colors.font};
+  display: flex;
+  align-items: center;
+  justify-content: center;
+`;
```

Comment:
Maybe you could put all different styled links in one file? `Links.js`?
I saw "LinkNoDeco" for example, is already existing which you could also put in this file.